resonantLimits/plot.py

```diff
@@ -197,63 +232,72 @@ def create_limits_plot(indirs, outfile, masses, labels, signal, period, varfit,
+        agraph["obs"][-1].SetMarkerStyle(20)
+        agraph["obs"][-1].SetFillStyle(0)
 
         if ndirs==1:
-            gr1sigma[-1].SetMarkerStyle(0)
-            gr1sigma[-1].SetMarkerColor(3)
-            gr1sigma[-1].SetFillColor(ROOT.kGreen+1)
-            gr1sigma[-1].SetLineColor(ROOT.kGreen+1)
-            gr1sigma[-1].SetFillStyle(1001)
+            agraph["sig1"][-1].SetMarkerStyle(0)
+            agraph["sig1"][-1].SetMarkerColor(3)
+            agraph["sig1"][-1].SetFillColor(ROOT.kGreen+1)
+            agraph["sig1"][-1].SetLineColor(ROOT.kGreen+1)
+            agraph["sig1"][-1].SetFillStyle(1001)
          
-            gr2sigma[-1].SetMarkerStyle(0)
-            gr2sigma[-1].SetMarkerColor(5)
-            gr2sigma[-1].SetFillColor(ROOT.kOrange)
-            gr2sigma[-1].SetLineColor(ROOT.kOrange)
-            gr2sigma[-1].SetFillStyle(1001)
-
-        if ndirs==1:
-           leg_labels = ('Observed', 'Median expected')
+            agraph["sig2"][-1].SetMarkerStyle(0)
+            agraph["sig2"][-1].SetMarkerColor(5)
+            agraph["sig2"][-1].SetFillColor(ROOT.kOrange)
+            agraph["sig2"][-1].SetLineColor(ROOT.kOrange)
+            agraph["sig2"][-1].SetFillStyle(1001)
+            
+            leg_labels = ('Observed', 'Median exp.')
         else:
-           leg_labels = ('Obs. ' + labels[2+idx][:20],
-                         'Exp. ' + labels[2+idx][:20])
-        #legend.AddEntry(grobs[-1], leg_labels[0], 'l')
-        legend.AddEntry(grexp[-1], leg_labels[1], 'l')
+            leg_labels = ('Obs. ' + labels[2+idx][:20],
+                          'Exp. ' + labels[2+idx][:20])
+
+        if plot_atlas and idx == 0:
+            agraph["atlas_nom"][-1].SetLineColor(39)
+            agraph["atlas_nom"][-1].SetLineWidth(3)
+            agraph["atlas_nom"][-1].SetLineStyle(2)
+            
+            agraph["atlas_sig1"][-1].SetMarkerStyle(0)
+            agraph["atlas_sig1"][-1].SetMarkerColor(3)
+            agraph["atlas_sig1"][-1].SetFillColor(18)
+            agraph["atlas_sig1"][-1].SetLineColor(18)
+            agraph["atlas_sig1"][-1].SetFillStyle(1001)
+
+        if plot_atlas and idx == 0:
+            legend.AddEntry(agraph["atlas_nom"][-1], "ATLAS Full Run2 exp.", 'l')
+            legend.AddEntry(agraph["atlas_sig1"][-1], 'ATLAS 68% exp.', 'f')
+        legend.AddEntry(agraph["exp"][-1], leg_labels[1], 'l')
         if ndirs==1:
-            legend.AddEntry(gr1sigma[-1], '68% expected', 'f')
-            legend.AddEntry(gr2sigma[-1], '95% expected', 'f')
-            gr2sigma[-1].Draw('3same')
-            gr1sigma[-1].Draw('3same')
-        grexp[-1].Draw('Lsame')
+            legend.AddEntry(agraph["sig1"][-1], '68% exp.', 'f')
+            legend.AddEntry(agraph["sig2"][-1], '95% exp.', 'f')
+            agraph["sig2"][-1].Draw('3same')
+            agraph["sig1"][-1].Draw('3same')
+        if plot_atlas and idx == 0:
+            agraph["atlas_sig1"][-1].Draw('3same')
```

Comment:
I would probably remove the 1 sigma band for ATLAS' result, but it's mostly a matter of taste